examples/v2/chat2.nim

```diff
@@ -158,7 +158,11 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
 
   # waitFor vs await
   await node.start()
-  await node.mountRelay(conf.topics.split(" "))
+
+  if conf.filternode != "":
```

Comment:
But wouldn't it require a node to also be able to run Relay and Filter, so that it can actual push messages from the Relay network to a Filter only node?

Reply:
We are starting a relay, we just aren't subscribing, so we can push but aren't receiving

Reply:
you like reviewing PRs after they are merged @kdeme! :D